src/netif.c

```diff
@@ -1602,7 +1602,7 @@ static int get_ipaddr6(const struct ip_setting_handler *handler, struct netif *n
           debug("[%s %d]: Address scope = '%s'", __FILE__, __LINE__, addr_scope);
         }
 
-        sprintf(prefix_str, "/%d", prefix);
+        snprintf(prefix_str, sizeof(prefix_str), "/%d", prefix);
```

Comment:
can put the magic number replacement  above instead of the sizeof...

Reply:
I prefer sizeof ;) rather than a precompiler macro

Reply:
corrected